src/Selenium2Library/keywords/_screenshot.py

```diff
@@ -17,49 +17,96 @@ def __init__(self):
     def set_screenshot_directory(self, path, persist=False):
         """Sets the root output directory for captured screenshots.
 
-        ``path`` argument specifies the absolute path where the screenshots should
-        be written to. If the specified ``path`` does not exist, it will be created.
-        Setting ``persist`` specifies that the given ``path`` should
-        be used for the rest of the test execution, otherwise the path will be restored
-        at the end of the currently executing scope.
+        ``path`` argument specifies the absolute path where the screenshots
+        should be written to. If the specified ``path`` does not exist,
+        it will be created. Setting ``persist`` specifies that the given
+        ``path`` should be used for the rest of the test execution, otherwise
+        the path will be restored at the end of the currently executing scope.
         """
         path = os.path.abspath(path)
         self._create_directory(path)
         if persist is False:
             self._screenshot_path_stack.append(self.screenshot_root_directory)
             # Restore after current scope ends
-            utils.events.on('scope_end', 'current', self._restore_screenshot_directory)
+            utils.events.on('scope_end', 'current',
+                            self._restore_screenshot_directory)
 
         self.screenshot_root_directory = path
 
-    def capture_page_screenshot(self, filename=None):
+    def capture_page_screenshot(
+            self, filename='selenium-screenshot-{index}.png'):
```

Comment:
This is one of the (PEP-8 recommendations)[https://www.python.org/dev/peps/pep-0008/#indentation]
```
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)
```
But I do not feel strongly about it, it complies to PEP-8 also when done like this:
```
def capture_page_screenshot(self,
                            filename='selenium-screenshot-{index}.png'):
```

Done